__tests__/fixtures/why/resolution/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "dependencies": {
+    "foo": "file:../basic"
+  },
+  "resolutions": {
+    "glob": "7.0.0"
```

Comment:
This will make sure version 7.0.0 is always installed, even though the `basic` fixture specifies 7.1.1, which makes this a good testing case.